common/models/wordbreakers/package.json

```diff
@@ -44,7 +44,8 @@
     "prepublish": "npm run build",
     "pretest": "npm run build",
     "test": "mocha",
-    "mocha": "mocha"
+    "mocha": "mocha",
+    "c8": "c8"
```

Comment:
Ditto for all these scripts (and probably remove prepublish and pretest scripts so that we aren't depending on those as side-effects)